proto/v1/rollout_service.proto

```diff
@@ -202,6 +202,16 @@ message Plan {
 
     // If RollbackEnabled, build the RollbackSheetID of the task.
     bool rollback_enabled = 5;
+
```

Comment:
If we have CreateRollout(), shall we have CreateRollback()?

Reply:
Maybe. Let's migrate first, then we add new stuff.